include/dlaf/matrix/tile.h

```diff
@@ -222,11 +223,9 @@ struct UnwrapFuture<hpx::future<Tile<T, D>>> {
 
 /// Callable object used for the unwrapExtendTiles function below.
 template <typename F>
-struct UnwrapExtendTiles {
-  F f;
-
+class UnwrapExtendTiles {
   template <typename... Ts>
-  auto operator()(Ts&&... ts) {
+  auto call_helper(std::true_type, Ts&&... ts) {
```

Comment:
According to the style:
```suggestion
  auto callHelper(std::true_type, Ts&&... ts) {
```